docs/user-manual/en/configuring-transports.md

```diff
@@ -239,6 +239,17 @@ Netty for simple TCP:
 -   `tcpReceiveBufferSize`. This parameter determines the size of the
     TCP receive buffer in bytes. The default value for this property is
     `32768` bytes (32KiB).
+    
+-   `nettyWriteBufferLowWaterMark`. This parameter determines the low water mark of 
+    the Netty write buffer. Once the number of bytes queued in the write buffer exceeded 
+    the high water mark and then dropped down below this value, Netty's channel 
+    will start to be writable again. The default value for this property is 
+    `32768` bytes (32KiB).
+ 
+-   `nettyWriteBufferHighWaterMark`. This parameter determines the high water mark of 
+    the Netty write buffer. If the number of bytes queued in the write buffer exceeds 
+    this value, Netty's channel will start to be not writable. The default value for 
+    this property is `131072` bytes (128KiB).
```

Comment:
Can you add a test where you validate this parameter through an URI?

Look for a test parsing the Connector and Acceptor, and validate if the value was parsed, please?



I will need this PR done ASAP, as I was doing something similar on my replication tests.